scripts/arango/arango-backup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+echo "making backup of arango data..."
+
+DATE=`date +"%Y-%m-%d--%H-%M-%S"`
+echo "Date: $DATE"
+
+arangodump \
+--output-directory "$ARANGO_DOCKER_VOLUME_SCRIPTS_DIR/database-backup/$DATE" \
```

Comment:
What happens if the `$ARANGO_DOCKER_VOLUME_SCRIPTS_DIR` environment variable is not set? I would provide a fallback directory or exit with an error here.